### ISSUE TITLE
[BUG] [Linter] False positiv ophan tag for option in datalist

### ISSUE THREAD

**Issue by henhuy** [djlint/djLint#822]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - Python Version 3.11.4
 - djLint Version 1.34.1
 - template language: django templating

## Issue
Actually linter and formatter are not detecting my template correctly.
First formatter tries to "fix" template, afterwards I'm getting following error:
`H025 6:36 Tag seems to be an orphan. <option value="{{ ab`
Maybe it cannot handle `datalist` ?

## How To Reproduce
I use following template:
```
<datalist id="abbreviations">
  {% for abb in abbreviation_list %}<option value="{{ abb }}">{% endfor %}
</datalist>
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```
blank_line_after_tag = "load,extends"
close_void_tags = true
format_css = true
format_js = true
# TODO: remove T002 when fixed https://github.com/Riverside-Healthcare/djLint/issues/687
ignore = "H006,H030,H031,T002"
include = "H017,H035"
indent = 2
max_line_length = 119
profile = "django"
```
